eotsmanager/localmanager.go

```diff
@@ -121,8 +123,10 @@ func (lm *LocalEOTSManager) CreateKeyWithMnemonic(name, mnemonic, passphrase str
 		return nil, err
 	}
 
+	// when the first key is created for the `file` keyring backend, it will prompt for a passphrase twice
```

Comment:
They call passwords in the variables, but passphrase in in the prompt https://github.com/cosmos/keyring/blob/master/file.go#L61

:exploding_head: